tests/foreman/destructive/test_capsule_loadbalancer.py

```diff
@@ -92,20 +109,20 @@ def setup_capsules(
 @pytest.fixture(scope='module')
 def setup_haproxy(
     module_org,
-    rhel7_contenthost_module,
+    module_rhel_contenthost,
     content_for_client,
     module_target_sat,
     setup_capsules,
 ):
     """Install and configure haproxy and setup logging"""
-    haproxy = rhel7_contenthost_module
+    haproxy = module_rhel_contenthost
     # Using same AK for haproxy just for packages
     haproxy_ak = content_for_client['client_ak']
     haproxy.execute('firewall-cmd --add-service RH-Satellite-6-capsule')
     haproxy.execute('firewall-cmd --runtime-to-permanent')
     haproxy.install_katello_ca(module_target_sat)
     haproxy.register_contenthost(module_org.label, haproxy_ak.name)
```

Comment:
I would just recommend to use global registration
```suggestion
    haproxy.register(module_org, None, haproxy_ak.name, module_target_sat)
```

Reply:
I'd say this is outside of the scope of the PR, but if other reviewers agree to change it now I'll update it.